var/spack/repos/builtin/packages/root/package.py

```diff
@@ -190,6 +190,7 @@ class Root(CMakePackage):
     variant("arrow", default=False, description="Enable Arrow interface")
     variant("cuda", when="@6.08.00:", default=False, description="Enable CUDA support")
     variant("cudnn", when="@6.20.02:", default=False, description="Enable cuDNN support")
+    variant("daos", default=False, description="Enable RNTuple support for DAOS storage")
```

Comment:
From when is this variant available?

Reply:
On master I assume.

Reply:
master for sure, but daos support showed up in .... wow, 2020.   Version 6.26 or newer
  ```
  $ git log cmake/modules/FindDAOS.cmake
commit d0d4ed5d0e3d6c9c32cf3477eef6e9ccd41f2678
Author: Javier Lopez-Gomez <javier.lopez.gomez@cern.ch>
Date:   Sun Nov 15 03:10:44 2020 +0100

    [ntuple] DAOS support: added libdaos C++ classes
$ git describe d0d4ed5d0e3d6c9c32cf3477eef6e9ccd41f2678
v6-25-01-831-gd0d4ed5d0e```

Reply:
So, `daos` support in spack was just added a lot later than `daos` support in ROOT :-) Well, we can certainly add it to the ROOT package.
```suggestion
    variant("daos", default=False, description="Enable RNTuple support for DAOS storage", when="@6:26:")
```